redirects.txt

```diff
@@ -0,0 +1,9 @@
+# Comment lines start with a hash (#) and are ignored
+# Each redirect should appear on its own line
+# Do not include a file extension
+# Paths don't need a slash in front of them
+# We are using the dirhtml builder so we need a trailing slash at the end
+# The paths should match what's in the build directory
+# Example:
+# redirect/from redirect/to
```

Comment:
The directions say a trailing slash is necessary at the end, which makes the example here a bit confusing.  I assume what it really meant to say is that a trailing slash is required "for directories"?

The example might be more examplly if it looked more like an actual path in our docs, e.g.:
```
# howto/ how-to/
# how-to/foo.rst how-to/bar.rst
```

Reply:
Thanks for picking this up - I very stupidly didn't include the trailing slashes in the example. If you put file extensions it will throw errors. Since we're using dirhtml to build, the built docs look like `/how-to/foo/index.html`, and since every "file" is index.html, you don't need to specify it. The file name is essentially nothing more than another directory. I'll tidy up the example and see if I can make this a bit more explicit to reduce potential for confusion.

Reply:
Looks great!